shared/productTypes.ts

```diff
@@ -591,6 +595,59 @@ export const PRODUCT_TYPES: { [productKey in ProductTypeKeys]: ProductType } = {
 			explicitSingleDayOfWeek: 'Friday',
 		},
 	},
+	tierthree: {
+		productTitle: () => 'Digital + Print',
+		friendlyName: () => 'digital + print subscription',
```

Comment:
the case of this doesn't seem to be consistent across products, I assume that's a style/branding thing?

Reply:
Perhaps separate from this pr we could do an audit/review of our use of product titles with UI/UX